### ISSUE TITLE
Problem using NTPq on Telegraf Windows

### ISSUE THREAD

**Issue by tastiestemperor** [influxdata/telegraf#5906]
I can't get the NTPq plugin to work with Telegraf Windows. Nor can I find any detailed instruction on how to install a plugin on Windows.

**Config**
```
# Get standard NTP query metrics, requires ntpq executable
[[inputs.ntpq]]
## If false, set the -n ntpq flag. Can reduce metric gather times.
dns_lookup = true
```

**Log**
```
Error in plugin: exec: "NTPq": executable file not found in %PATH%
```

**Steps taken**
Installed Telegraf as Windows service as stated on the GitHub page.
Added the plugin configuration to the Telegraf configuration under INPUT
Added the NTPq GO file in the Telegraf path "C:/Program Files/Telegraf" (Not sure if that's correct)
Restarted the Telegraf service

**Main questions**
How are you supposed to install a Telegraf plugin in Windows?
Is the path correct for the GO file or do I need to define the correct path in the configuration file?

**Comment by danielnelson** [comment]
I haven't ran this plugin on Windows, but I'm not sure I follow what you have tried, what "NTPq GO file" are you referring to?

**Comment by tastiestemperor** [comment]
> I haven't ran this plugin on Windows, but I'm not sure I follow what you have tried, what "NTPq GO file" are you referring to?

The NTPq file I mentioned was is just the GO-script that can be found [Here](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/ntpql)

That being said our inhouse documentation might be very very outdated. I reread a few documents from this GitHub and even the documentation from Telegraf itself. And if I;m correct you only need to add the configuration to the Telegraf config file under "INPUT".

If you could point me into the right direction on how to enable/install Telegraf plugins on Win Os. Cause I'm clueless atm.

Also I think I figured out what the error was refering to, the missing NTPq executable. Where can I find the required executable? 
`Get standard NTP query metrics, requires ntpq executable.`

**Comment by danielnelson** [comment]
To enable/disable a plugin, you just need to add/remove the definition from your Telegraf configuration file.
```toml
[[inputs.ntpq]]
  dns_lookup = true
```

It sounds like you already have this enabled though, so you just need to add the `ntpq.exe` binary to your PATH.

There are at least 2 ntp projects that work with this plugin: www.ntp.org and www.ntpsec.org.

I haven't tried either of these implementations on Windows, but I found this link for the ntp.org project:
http://support.ntp.org/bin/view/Main/ExternalTimeRelatedLinks#Microsoft_Windows

If you decided to try ntpsec, I believe you will need to build from source:
https://docs.ntpsec.org/latest/build.html#build

**Comment by danielnelson** [comment]
Closing since this is a setup question, but would be interested in hearing back if you are able to get the plugin working on Windows and what is required.

**Comment by tastiestemperor** [comment]
@danielnelson I can't tell you how happy I'm you saved my project and all. I got it working this Sunday and it was really simple. I should have read more documentation before asking for help haha.

I used the NTP-port made by Meinberg which is far more accurate then our standard solution. The installation of the NTP-port if fairly straight forward and with the configuration of the NTPq plugin it worked straight out of the box.

**Setup steps** _(This is what I used, which worked for my use case)_
1. Install NTP by Meinberg ([Link](https://www.meinbergglobal.com/english/sw/ntp.htm))
2. Configure NTP software (Selecting the NTP pool and setting up the local account)
3. Add the NTPq executable _(Found in the installation directory)_ to the Telegraf Path. (Shortcut is also possible)
4. Add the NTPq configuration to the Telegraf Input config
5. Restart the services of the NTP-software and Telegraf (A reboot is preferred imo)
5.1 (Optional) Verify that the NTP-software works by using a sample command ex. NTPq -pn in CMD
6. Verify that the NTPQ data is being writing to the Influxdb (This should be no problem)
7. (Optional) Configure dashboard to needs with graphs showing the offset/delay.
